src/Scene.js

```diff
@@ -365,6 +367,24 @@ class Scene extends React.Component<SceneProps, {}> {
                             {grid}
                             <g clipPath='url(#Scene-clippath)'>
                                 {this.drawCharacterPath()}
+                                <rect
+                                    // Dotted rectangle to indicate starting grid cell
```

Comment:
Maybe something more general like: "Cell outline to indicate starting grid cell"